File: src/main/java/com/uwetrottmann/trakt/v2/entities/Episode.java
package com.uwetrottmann.trakt.v2.entities;

import org.joda.time.DateTime;

public class Episode extends BaseEntity {

    public Integer season;
    public Integer number;
    public EpisodeIds ids;

    // extended info

    public Integer number_abs;
    public String overview;
    public DateTime first_aired;
    public Double rating;

}

File: src/main/kotlin/com/redissi/trakt/entities/Episode.kt
package com.redissi.trakt.entities

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import java.time.OffsetDateTime

@JsonClass(generateAdapter = true)
class Episode(
    override val title: String? = null,
    override val overview: String? = null,
    override val rating: Double? = null,
    override val votes: Int? = null,
    override val updatedAt: OffsetDateTime? = null,
    override val availableTranslations: List<String>? = null,
    val season: Int? = null,
    val number: Int? = null,
    val ids: EpisodeIds? = null,
    // extended info
    @Json(name = "number_abs")
    val number_abs: Int? = null,
    @Json(name = "first_aired")
    val firstAired: OffsetDateTime? = null,
    @Json(name = "comment_count")
    val commentCount: Int? = null,
    val runtime: Int? = null
) : BaseEntity()